app/server/models/custom_attribute_user_storage.py

```diff
@@ -0,0 +1,13 @@
+from server import db
+from server.models.utils import ModelBase
+from server.models.user import User
+
+
+class CustomAttributeUserStorage(ModelBase):
```

Comment:
you'll want to add this to `app/migrations/env.py` so it gets picked up in the migration generation. still hoping we can figure out a better way to do that, very easy to forget to add new files to there

Reply:
Thanks!